### PR TITLE
Release 3.2.0

### PR DESCRIPTION
**Pull request by FinalAngel**
- added yuidocs syntax and generators
- added Cl.Utils.redirectTo
- added Cl.Utils.getStorage and Cl.Utils.setStorage
- deprecate Cl.Utils.mobile and Cl.utils.tablet
- update npm dependencies
- update CKEditor setting styles and removed double usage
- fixes an issue with navigation.html

Fixes Issues:
#61 
#27 
#56

### CODE REVIEW

File: docs/general/best-practices.rst [hunk 1]
```diff
@@ -65,6 +66,12 @@ We are implementing the following standard libraries in addition to the default
 We implemented `Bower <http://bower.io/>`_ to help you manage dependencies. Packages are automatically downloaded into
 ``/static/vendor/`` but **not** moved to their appropriate folders. This still requires manual work.
 
+Documentation
+-------------
+
+We write API documentation using the `YUIDoc <http://yui.github.io/yuidoc/>`_ syntax. For HTML and CSS please refer
+to the styleguide.
```
Comment (Brankub) [inline docs/general/best-practices.rst]: unnecessary break line or not?

Comment (FinalAngel) [inline docs/general/best-practices.rst]: we need to stay within the 120 line limit

File: templates/includes/menu/navigation.html [hunk 2]
```diff
@@ -3,7 +3,10 @@
 
 {% for child in children %}
 <li class="child{% if child.ancestor %} ancestor{% endif %}{% if child.selected %} active{% endif %}{% if child.children and dropdown %} children dropdown{% endif %}{% if forloop.first %} first{% endif %}{% if forloop.last %} last{% endif %}">
-    <a href="{{ child.get_absolute_url }}"{% if child.children and not child.parent and dropdown %} class="dropdown-toggle" data-toggle="dropdown"{% endif %}><span>{{ child.get_menu_title }}</span>{% if child.children and not child.parent and dropdown %} <span class="caret"></span>{% endif %}</a>{% if child.children %}
+    <a href="{{ child.get_absolute_url }}"{% if child.children and not child.parent and dropdown %} class="dropdown-toggle" data-toggle="dropdown"{% endif %}>
+        <span>{{ child.get_menu_title }}
+        {% if child.children and not child.parent and dropdown %} <span class="caret"></span>{% endif %}</span>
+    </a>{% if child.children %}
```
Comment (sergii-kulakovskyi) [inline templates/includes/menu/navigation.html]: maybe it'd be better to move if statement to the next line

File: docs/general/best-practices.rst (hunk 1 above)
Comment (vxsx) [inline docs/general/best-practices.rst]: maybe it makes sense to do linebreak after the dot?

File: templates/includes/menu/navigation.html (hunk 2 above)
Comment (sergii-kulakovskyi) [inline templates/includes/menu/navigation.html]: I'd do like this
```
<li class="child{% if child.ancestor %} ancestor{% endif %}{% if child.selected %} active{% endif %}{% if child.children and dropdown %} children dropdown{% endif %}{% if forloop.first %} first{% endif %}{% if forloop.last %} last{% endif %}">
    <a href="{{ child.get_absolute_url }}"{% if child.children and not child.parent and dropdown %} class="dropdown-toggle" data-toggle="dropdown"{% endif %}>
        <span>
            {{ child.get_menu_title }}
            {% if child.children and not child.parent and dropdown %}<span class="caret"></span>{% endif %}
        </span>
    </a>
    {% if child.children %}
        <ul{% if child.children and not child.parent and dropdown %} class="dropdown-menu" role="menu"{% endif %}>
            {% show_menu from_level to_level extra_inactive extra_active template "" "" child %}
        </ul>
    {% endif %}
</li>
```

File: docs/general/best-practices.rst (hunk 1 above)
Comment (FinalAngel) [inline docs/general/best-practices.rst]: there is a linebreak ;)

File: templates/includes/menu/navigation.html (hunk 2 above)
Comment (FinalAngel) [inline templates/includes/menu/navigation.html]: correct, will adapt

**Comment by sergii-kulakovskyi** [comment]
:+1:

File: static/js/addons/cl.utils.js [hunk 3]
```diff
@@ -48,16 +64,40 @@ var Cl = window.Cl || {};
...
-        // INFO: add here more Utilities required for the project
+        /**
+         * Save information within local storage
+         *
+         * @method setStorage
+         * @param token {String} namespace
+         * @param value {String} storage value
+         */
+        setStorage: function (token, value) {
+            if (window.localStorage) {
```
Comment (Flight) [inline static/js/addons/cl.utils.js]: Error here
http://stackoverflow.com/questions/11214404/how-to-detect-if-browser-supports-html5-local-storage

**Pull request merged**